src/site/content/en/learn/accessibility/javascript/index.md

```diff
@@ -3,10 +3,216 @@ title: 'JavaScript'
 authors:
   - cariefisher
 description: Write accessible JavaScript.
-date: 2022-09-21
+date: 2022-12-02
 placeholder: true
```

Comment:
```suggestion
placeholder: false
```

@AaronForinton fyi for future posts, placeholder needs to be false to add the updates to the navigation!